pastebin.cr

```diff
@@ -13,10 +12,8 @@ server = HTTP::Server.new do |context|
       File.open("files/#{generated_name}", "w") { |f| IO.copy(part.body, f) }
       context.response << generated_name
     end
-  when "GET"
-    File.open("files/#{context.request.path.split("/")[1]}") do |f|
-      IO.copy f, context.response.output
-    end
+  elsif context.request.method == "GET"
+    File.open("files/#{context.request.path.split('/')[1]}") { |f| IO.copy f, context.response.output }
```

Comment:
I would like to keep all the lines less than 80 characters, we could do it if we moved the file path to a variable, like so:
```suggestion
    path = "files/#{context.request.path.split('/')[1]}"
    File.open(path) { |f| IO.copy f, context.response.output }
```
It's longer, but easier on my eyes